Add man page for condor_token_*. #7361 (#83)
htcondor/htcondor: docs/man-pages/condor_token_request_auto_approve.rst

*condor_token_request_auto_approve*
===================================

generate a new rule to automatically approve token requests
:index:`condor_token_request_auto_approve<single: condor_token_request_auto_approve; HTCondor commands>`\ :index:`condor_token_request_auto_approve command`

Synopsis
--------

**condor_token_request_auto_approve** **-netblock** *network* **-lifetime** *val*
[**-pool** *pool_name*] [**-name** hostname] [**-type** *type*]
[**-debug**]

> **condor_token_request** [**-help** ]

Reviewer: ```suggestion
**condor_token_request_auto_approve** [**-help** ]
```

Description
-----------

> *condor_token_request_auto_approve* will install an auto-approval rule for token requests.

Reviewer: ```suggestion
*condor_token_request_auto_approve* will install a temporary auto-approval rule for token requests.
```

Any token request matching the auto-approval rule will be immediately approved instead
of requiring administrator approval

Automatic request approval is intended to help administrators initially setup their cluster.
To install a new rule, you must specify both a network and a lifetime; requests are only
approved if they come from that given source network, are within the rule lifetime, are
limited to ``ADVERTISE_SCHEDD`` or ``ADVERTISE_STARTD`` permissions, and are for the
``condor`` identity.  When a *condor_startd* or *condor_schedd* is started and cannot
communicate with the collector, they will automatically generate token requests that meet
the last two conditions.

It is not safe to enable auto-approval when users have access to any of the involved hosts
or networks.

> To remove auto-approval rules, simply run ``condor_reconfig`` against the remote daemon.:

Reviewer: ```suggestion
To remove auto-approval rules, run ``condor_reconfig`` against the remote daemon.
```
